content/src/content/jcr_root/apps/core/wcm/sandbox/components/teaser/v1/teaser/teaser.html

```diff
@@ -20,9 +20,11 @@
 
     <div class="cmp-teaser__image" data-sly-resource="${teaser.imageResource}"></div>
 
-    <h2 class="cmp-teaser__title" data-sly-test.title="${teaser.title}">${title}</h2>
-    <h3 class="cmp-teaser__description" data-sly-test.description="${teaser.description}">${description}</h3>
-
-    <a class="cmp-teaser__link" href="${teaser.linkURL}">${teaser.linkText}</a>
```

Comment:
What is the reasoning behind removing the anchor 'cmp-teaser__link' element completely? Some customers would appreciate a separate anchor tag that can be configured to look like a Call To Action button i.e: https://getbootstrap.com/docs/4.0/components/card/ ?

Reply:
@godanny86 We plan to add the possibility of having CTAs as a second stage, in a few days.

Reply:
great!